clients/egui/src/account/suggested_docs.rs

```diff
@@ -0,0 +1,131 @@
+use std::sync::Arc;
+
+use eframe::egui;
+
+use crate::model::DocType;
+
+pub struct SuggestedDocs {
+    files: Vec<SuggestedFile>,
+}
+struct SuggestedFile {
+    name: String,
+    path: String,
+    id: lb::Uuid,
+}
+
+impl SuggestedDocs {
+    pub fn new(core: &Arc<lb::Core>) -> Self {
+        let files = core
+            .suggested_docs(lb::RankingWeights::default())
+            .unwrap_or_default()
+            .iter()
+            .filter_map(|id| {
+                let file = core.get_file_by_id(*id);
+                if file.is_err() {
+                    return None;
+                };
+                let path = core.get_path_by_id(*id).unwrap_or_default();
+
+                Some(SuggestedFile { name: file.unwrap().name, path, id: *id })
```

Comment:
I'm skeptical of all these `unwrap_or_default`s as well as the straight up `unwrap`